x/blob/types/blob_tx.go

```diff
@@ -102,12 +103,12 @@ func ValidateBlobTx(txcfg client.TxEncodingConfig, bTx *tx.BlobTx, subtreeRootTh
 	}
 
 	// verify that the commitment of the blob matches that of the msgPFB
+	calculatedCommitments, err := inclusion.CreateParallelCommitments(bTx.Blobs, merkle.HashFromByteSlices, subtreeRootThreshold, runtime.NumCPU()*4)
```

Comment:
Maybe try also with `ParallelHashFromByteSlices` and see the performance difference

Reply:
With 92 byte roots (32 + 29 * 2) parallel computations doesn't give us any benefit even for 10k hashes (according to my measurements), in reality with 32mb blob sizes we have less than 200 subtree roots per blob, so it seems we shouldn't use parallel function here.